### DISCUSSION TITLE
Is there a way to know the download setup for ungoogled chromium is unmodified?

### DISCUSSION THREAD

**Discussion by bluecheeser** [Eloston/ungoogled-chromium#1466] [Q&A]
Is there some way to check the the downloaded file for tampering?

**Comment by Eloston** [comment]
It depends where you downloaded ungoogled-chromium, and what kind of tamper detection you are looking for.

- If you got it from https://ungoogled-software.github.io/ungoogled-chromium-binaries/, then there are file hashes that will ensure the uploader hasn't tampered with the binary since uploading.
- For Debian-based systems with OBS binaries, such binaries are built, signed, and stored on OBS servers. So the only tamper detection you can get is between the OBS servers and your machine.
- If you want to verify that the binary comes from the source code posted here on GitHub, this is a much more difficult problem that can be addressed by reproducible builds. There is an issue that tracks this, but there is a lot more work to make it reliable and trustworthy.

**Reply by bluecheeser** [reply]
I use ungoogled updater from https://github.com/iklevente/ChromiumForWindows which takes UC form https://github.com/macchrome/winchrome/ is this trusted or is there a way to check?
